scripts/start-bridging-environment.js

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env node
+
+/*
+ * Start the Reputation Miner/Oracle for local development
+ * DO NOT USE IN PRODUCTION
+ */
+
+const path = require("path");
+const ethers = require("ethers");
+const { TruffleLoader } = require("../packages/package-utils");
+const { WAD } = require("../helpers/constants");
+
+const loader = new TruffleLoader({
+  contractDir: path.resolve(__dirname, "..", "build", "contracts"),
+});
+
+const ADDRESS_ZERO = ethers.constants.AddressZero;
+const ethersForeignProvider = new ethers.providers.JsonRpcProvider("http://127.0.0.1:8546");
+const ethersForeignSigner = new ethers.providers.JsonRpcProvider("http://127.0.0.1:8546").getSigner();
+const ethersHomeSigner = new ethers.providers.JsonRpcProvider("http://127.0.0.1:8545").getSigner();
+// const ethersHomeProvider = new ethers.providers.JsonRpcProvider("http://127.0.0.1:8545");
+const BridgeMonitor = require("./bridgeMonitor");
+
+async function start() {
+  const accounts = await getGanacheAccounts();
+  const GnosisSafeProxyFactory = await loader.load({ contractName: "GnosisSafeProxyFactory" }, { abi: true, address: false });
+  const GnosisSafe = await loader.load({ contractName: "GnosisSafe" }, { abi: true, address: false });
+  const ZodiacBridgeModuleMock = await loader.load({ contractName: "ZodiacBridgeModuleMock" }, { abi: true, address: false });
+  const BridgeMock = await loader.load({ contractName: "BridgeMock" }, { abi: true, address: false });
+  const erc721Mock = await loader.load({ contractName: "ERC721Mock" }, { abi: true, address: false });
+  const TokenContract = await loader.load({ contractName: "Token" }, { abi: true, address: false });
+
+  const gspf = await new ethers.Contract("0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2", GnosisSafeProxyFactory.abi, ethersForeignSigner);
+
+  const receipt = await gspf.createProxy("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", "0x");
+  let tx = await receipt.wait();
+  console.log(tx.events[0]);
+
+  const safeAddress = tx.events[0].args.proxy;
+  const gs = await new ethers.Contract(safeAddress, GnosisSafe.abi, ethersForeignSigner);
```

Comment:
I don't love these short variable names. My preference would be `gnosisSafe` to denote specific instances, and `GnosisSafe` to denote the contract object.